packages/vite/src/node/config.ts

```diff
@@ -342,6 +342,8 @@ export type ResolvedConfig = Readonly<
     /** @internal */
     mainConfig: ResolvedConfig | null
     isProduction: boolean
+    /** @internal */
+    envDir: string
```

Comment:
I think it's safe enough to not need to make this internal. Currently, `ResolvedConfig` has `envDir?: string | undefined`, this would make a previous public API internal, which is sort-of a breaking change.

Removing the `/** @internal */` comment should be good.
```suggestion
    envDir: string
```

Reply:
Updated. nice catch. I feel even more that we need some type checking (maybe leverage `tsd`). I'll give it a try.